contract/src/lib.rs

```diff
@@ -238,8 +238,7 @@ impl BtcLightClient {
     ///
     /// # Panics
     /// Multiple cases
-    #[allow(clippy::needless_pass_by_value)]
-    #[pause(except(roles(Role::UnrestrictedVerifyTransaction)))]
```

Comment:
We also should remove this Role

Reply:
Removed 
https://github.com/Near-One/btc-light-client-contract/pull/71/commits/5aec461a5205321d8baaa853e6d6d12b505682ea

Also added a build script 
https://github.com/Near-One/btc-light-client-contract/pull/71/commits/853222c4a74f697a27877d84a6761dbef727576b